flair/datasets/sequence_labeling.py

```diff
@@ -1597,3 +1597,97 @@ def _download_wikiner(language_code: str, dataset_name: str):
                 words = line.split(" ")
                 for word in words:
                     out.write("\t".join(word.split("|")) + "\n")
+
+class UP_GERMAN(ColumnCorpus):
+    def __init__(
+            self,
+            base_path: Union[str, Path] = None,
+            in_memory: bool = True,
+            document_as_sequence: bool = False,
+    ):
+        """
+        Initialize the German dataset from the Universal Propositions Bank, comming from that webpage:
+        https://github.com/System-T/UniversalPropositions.
+
+        :param base_path: Default is None, meaning that corpus gets auto-downloaded and loaded. You can override this
+        to point to a different folder but typically this should not be necessary.
+        :param in_memory: If True, keeps dataset in memory giving speedups in training.
+        :param document_as_sequence: If True, all sentences of a document are read into a single Sentence object
+        """
+        if type(base_path) == str:
+            base_path: Path = Path(base_path)
+
+        # column format
+        columns = {1: "text", 10: "frame"}
```

Comment:
the column are wrong, it seems that column 10 is not the frame information

Reply:
It was column nr. 9, starting with 0 to count, sorry, my mistake. It got fixed